test/shared/src/test/scala/monocle/refined/StringsSpec.scala

```diff
@@ -0,0 +1,23 @@
+package monocle.refined
+
+import monocle.MonocleSuite
+import monocle.law.discipline.PrismTests
+import org.scalacheck.Cogen
+import eu.timepit.refined.scalacheck.string.startsWithArbitrary
+import eu.timepit.refined._
+
+
+import scalaz.Equal
+
+
+class StringsSpec extends MonocleSuite {
+
+  implicit val startsWithCoGen: Cogen[StartsWithString[W.`"hello"`.T]] = Cogen[String].contramap[StartsWithString[W.`"hello"`.T]](_.value)
```

Comment:
I think this could be added to refined-scalacheck for all refined types as:
```scala
implicit def refTypeCogen[F[_, _], T: Cogen, P](implicit rt: RefType[F]): Cogen[F[T, P]] =
    Cogen[T].contramap(tp => rt.unwrap(tp))
```
`RefType` is a type class that abstracts over `Refined` and other tag types so we wouldn't have to define `Cogen` instances for specific refined types anymore.